bundles/org.openhab.binding.lametrictime/src/3rdparty/java/org/openhab/binding/lametrictime/api/local/impl/LaMetricTimeLocalImpl.java

```diff
@@ -252,8 +256,11 @@ public void updateApplication(String packageName, String accessToken, WidgetUpda
         Response response = getClient().target(getApi().getEndpoints().getAppsListUrl())
                 .request(MediaType.APPLICATION_JSON_TYPE).get();
 
+        String entity = new BufferedReader(new InputStreamReader((InputStream) response.getEntity())).lines()
+                .collect(Collectors.joining("\n"));
```

Comment:
You should specify the charset to use in the InputStreamReader. The charset should be part of the content type of the response right?

Also, you should probably just input the reader directly into `fromJson` so that you don't have to go through the extra step of creating a string.

Reply:
Thanks for your (as always) good and helpful review comments! I've addressed this now.